packages/events/EventSystemFlags.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export type EventSystemFlags = number;
+
+export const PLUGIN_EVENT_SYSTEM = 1;
+export const RESPONDER_EVENT_SYSTEM = 1 << 1;
+export const IS_PASSIVE = 1 << 2;
+export const IS_ACTIVE = 1 << 3;
+export const PASSIVE_NOT_SUPPORTED = 1 << 4;
```

Comment:
why we don't use literal number here just like `maxSigned31BitInt`?

Reply:
These are used to do bitwise operations on, as there can be a combination of the above being used.

Reply:
I think the question was why `1 << 4` is used over `16`. `1 << 4` makes the intent clearer. `maxSigned31BitInt` "solves" this by adding a comment. I don't think it matters what pattern is used.

Reply:
yea, just like @eps1lon said

Reply:
This pattern is clear, as we may add more in the future, which is simply `1 << 5`, `1 << 6` etc. We use this approach throughout the React codebase and we've verified that Google Closure Compiler simplifies the constant value.